### DISCUSSION TITLE
Recent QSO table

### DISCUSSION THREAD

**Discussion by hairlesshobo** [magicbug/Cloudlog#767] [Ideas / Feature Requests]
I was wondering if Cloudlog already has some sort of capability to have a "recent QSO table" that could be placed on another web page in an iframe. I know the big providers like QRZ, Clublog, etc all tend to have one of these, was wondering if there is one in cloudlog and I have just missed it. 

If one does not exist, would you be open to a pull request for this functionality if I get around to adding it myself?

**Comment by magicbug** [comment]
/widgets/qsos if memory serves correctly at midnight.

**Comment by hairlesshobo** [comment]
Ah! I had a feeling it was already there and I just missed it. Thanks!

**Reply by magicbug** [reply]
I'm not actually sure its documented!

**Comment by hairlesshobo** [comment]
I would like to add a bit of customisation ability to it, via query string, such as the ability to specify the number of recent QSOs, flag to hide time, flag to not make callsigns a link, etc. Would you be open to a pull request for that?

**Reply by magicbug** [reply]
No objection, as long as it's backwards compatible

**Comment by pa8s** [comment]
Currently the output of the dates in the widgets/qsos list is dd/mm/yy although my cloudlog date format settings is yyyy-mm-dd. 
Could it be possible for this output to follow the set date format? I.e. the widgets/qsos dates will be displayed as yyyy-mm-dd, in my case.

**Comment by petrkr** [comment]
I was thinking about this too.. Seems widgets/qsos already not works at new 2.6.15 version
